frontend/packages/dev-console/src/components/import/DeployImage.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import { history } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { DeployImageFormData } from './import-types';
+import { createResources } from './deployImage-submit-utils';
+import { deployValidationSchema } from './deployImage-validation-utils';
+import DeployImageForm from './DeployImageForm';
+
+export interface DeployImageProps {
+  namespace: string;
+}
+
+const DeployImage: React.FC<DeployImageProps> = ({ namespace }) => {
+  const initialValues: DeployImageFormData = {
+    project: {
+      name: namespace || '',
+    },
+    application: {
+      name: '',
+      selectedKey: '',
+    },
+    name: '',
+    searchTerm: '',
+    isi: {
+      name: '',
+      image: {},
+      tag: '',
+      status: '',
+      ports: [],
+    },
+    image: {
```

Comment:
Because `CreateRoute.tsx` is looking for `image`. I didn't want to duplicate the route components so I just added the `image` object. 
Our current shared components are very specific to Import. 
AND `image` and `isi` share some fields, they are not the same. We could merge them. Any other ideas?

Reply:
At this point, I think this is something that can be cleaned up later. The forms need some refactoring to make them more modular and this fits under that.